### PR TITLE
Added Floyd and warshall algorithms' code

### LINKED ISSUE #331: [Feature Request] Adding Floyd and Warshall Algorithm code
## 🚀 Feature Request
Addition of  Floyd and Warshall Algorithm code

### Describe the solution you'd like
I would like to add the code for the Floyd and Warshall Algorithm.

---

**Would you like to work on this feature?**  
- [ * ] Yes  

Please assign this under the appropriate labels.

### PR DESCRIPTION
**Pull request by 1-SubhamSingh**
### Description
Added Floyd and warshall algorithms' code

Fixes #331

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ * ] Documentation update

### Checklist:

- [ * ] My code follows the style guidelines of this project
- [ * ] I have performed a self-review of my code
- [ * ] I have commented my code, particularly in hard-to-understand areas
- [ * ] I have added tests that prove my fix is effective or that my feature works
- [ * ] New and existing unit tests pass locally with my changes
- [ * ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/graphs/Floyd's_algorithm.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+id: flyod's_algo
```
Comment (ajay-dhangar) [inline docs/graphs/Floyd's_algorithm.md]: replace `id: flyod's_algo` with `id: flyod-algo`

File: docs/graphs/Warshall's-algo.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                }
+ printf("Transitive Closure of the given graph is:\n");
+                 for (i = 0; i < n; i++) {
+                    for (j = 0; j < n; j++) {
+                        printf("%d\t", T[i][j]);
+                        }
+                    printf("\n");
+                    }
+                return 0;
+}
```
Comment (ajay-dhangar) [inline docs/graphs/Warshall's-algo.md:60]: close your code with ```` ``` ````

File: docs/graphs/Warshall's-algo.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+id: warshall's_algo
```
Comment (ajay-dhangar) [inline docs/graphs/Warshall's-algo.md]: replace `id: warshall's_algo` with `id: warshall-algo`

File: docs/graphs/Floyd's_algorithm.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+ printf("Solution to All Pairs Shortest Distance Problem\n");
+ printf("Using Floyd's Algorithm - Distance Matrix of the given graph is: \n");
+ for (i = 0; i < n; i++) {
+ for (j = 0; j < n; j++) {
+ printf("%d\t", D[i][j]);
+ }
+ printf("\n");
+ }
+ return 0;
+}
```
Comment (ajay-dhangar) [inline docs/graphs/Floyd's_algorithm.md:64]: close your code with ```` ``` ````

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by 1-SubhamSingh** [comment]
@ajay-dhangar  I have made the changes , you can check.

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**